docusaurus-plugin-moonwave/src/components/LuaClass.js

```diff
@@ -151,6 +154,8 @@ export default function LuaClass({
 
   const luaClass = { ...rawLuaClass }
 
+  const typeLinks = new Map(preMapTypeLinks)
```

Comment:
What do we get out of making this a map? I would probably just keep it as an object.

To check if a key is in an object, all we need to do is use `in`, like `if (key in typeLinks)`